Dockerfile

```diff
@@ -1,15 +1,13 @@
-FROM registry.access.redhat.com/ubi8/nodejs-16
+FROM registry.access.redhat.com/ubi8/nodejs-18
 
-USER 0
+USER 1001
```

Comment:
**To the reviewer:**

This is intentional as a modern version of NPM would throw an error here since the **.npm** directory and the **.npmrc** file were previously owned by the root user. The error is as follows:

```
npm ERR! code EACCES
npm ERR! syscall mkdir
npm ERR! path /opt/app-root/src/.npm/_cacache
npm ERR! errno -13
npm ERR! 
npm ERR! Your cache folder contains root-owned files, due to a bug in
npm ERR! previous versions of npm which has since been addressed.
npm ERR! 
npm ERR! To permanently fix this problem, please run:
npm ERR!   sudo chown -R 1001:0 "/opt/app-root/src/.npm"
```

Reply:
ubi image should already use `1001` as current user by default, but to make it explicitly here is also good

Reply:
@hemslo, I know. This is purely for visibility.